egs2/wsj0_2mix/mixit_enh1/local/data.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# Copyright 2022  Carnegie Mellon University (Authors: Xuankai Chang)
+# Apache 2.0
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+function check_sorted {
+  file=$1
+  sort -k1,1 -u <$file >$file.tmp
+  if ! cmp -s $file $file.tmp; then
+    echo "$0: file $1 is not in sorted order or not unique, sorting it"
+    mv $file.tmp $file
+  else
+    rm $file.tmp
+  fi
+}
+
+help_message=$(cat << EOF
+Usage: $0 [--min_or_max <min/max>] [--sample_rate <8k/16k>]
+  optional argument:
+    [--min_or_max]: min (Default), max
+    [--sample_rate]: 8k (Default), 16k
+EOF
+)
+
+
+min_or_max=min
+sample_rate=8k
+
+local_data_args="$*"
+
+
+. utils/parse_options.sh
+
+if [ $# -ne 0 ]; then
+    echo "${help_message}"
+    exit 1;
+fi
+
+local/data_supervised.sh ${local_data_args}
+
+sup_train_set="tr_"${min_or_max}_${sample_rate}
+train_set="tr_"${min_or_max}_${sample_rate}_w_1spk_utt
```

Comment:
What is the meaning of the prefix `_w_1spk_utt`?

Reply:
The suffix means "with 1 speaker utterances".